From 642fe241551743813ee5f8b609468e0d13f0d504 Mon Sep 17 00:00:00 2001
From: Yann Gautier <yann.gautier@foss.st.com>
Date: Tue, 7 Mar 2023 10:55:56 +0100
Subject: [PATCH 288/954] docs(changelog): add st-rif scope

The st-rif will be used for ST drivers for Resource Isolation Framework.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 9f2ec8e32deca9b5e2af874ee541ae4d33d3314c docs(changelog): add st-rif scope]
Signed-off-by: Yann Gautier <yann.gautier@foss.st.com>
Change-Id: I09cc3a163a13149d9062e0a9ae1d573fc2830f83
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/294080
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Lionel DEBIEVE <lionel.debieve@foss.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 changelog.yaml | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/changelog.yaml b/changelog.yaml
index 1b8f193f74..ab6d2e5000 100644
--- a/changelog.yaml
+++ b/changelog.yaml
@@ -1147,6 +1147,9 @@ subsections:
           - title: Reset
             scope: st-reset
 
+          - title: Resource Isolation Framework
+            scope: st-rif
+
           - title: SDMMC2
             scope: st-sdmmc2
 
-- 
2.53.0

